CMakeLists.txt

```diff
@@ -240,6 +240,12 @@ add_compile_options(
   "-std=c++11"
 )
 
+if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(
```

Comment:
As far as I understand, gcc does not support this option. I could not find it here:
https://gcc.gnu.org/onlinedocs/gcc/Option-Summary.html